Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -6843,8 +6803,8 @@ def test_iterator__empty_page(self, mocker):
         results = []
         for res in search_indicators:
             results.append(res)
-        assert len(results) == 0
-        assert search_indicators.page == 19
+        assert len(results) == 7
```

Comment:
@merit-maita please remove the test, using the `page` search should no longer be supported after 6.2.